startup/31-xspress3.py

```diff
@@ -0,0 +1,355 @@
+print(f'Loading {__file__}...')
+
+
+import h5py
+import sys
+import time as ttime
+from ophyd.areadetector.plugins import PluginBase
+from ophyd import Signal, DeviceStatus
+from ophyd import Component as C
+from enum import Enum
+
+from hxntools.detectors.xspress3 import (XspressTrigger, Xspress3Detector,
+                                         Xspress3Channel, Xspress3FileStore)
```

Comment:
Once https://github.com/NSLS-II/nslsii/pull/79 is merged, we can use these classes from `nslsii`.